### ISSUE TITLE
Better Optimize Gitcoin for Both Mobile/Desktop

### ISSUE THREAD

**Issue by mkosowsk** [gitcoinco/web#592]
Note: this is possibly blocked by work on https://github.com/gitcoinco/web/issues/557 and/or https://github.com/gitcoinco/web/issues/560

It might make sense to revisit this once one or both of those issues are addressed.

Per Google PageSpeed: https://developers.google.com/speed/pagespeed/insights/

<img width="490" alt="screen shot 2018-03-11 at 7 40 03 am" src="https://user-images.githubusercontent.com/7970539/37279244-98e4a710-25c0-11e8-80a2-d9c217596b9f.png">

<img width="491" alt="screen shot 2018-03-11 at 7 40 09 am" src="https://user-images.githubusercontent.com/7970539/37279256-9e33c110-25c0-11e8-891f-6a3b76ebd32b.png">

Mobile is somewhat decent but desktop could use some work. I think for a first-pass it would be appropriate to target 90/100 for a mobile bounty and do 75/100 for a desktop bounty. It's nice to have these targets a la code coverage bounties.

Here's a good article for beginner techniques on optimization, think we could probably use some of the low hanging fruit here:

https://medium.freecodecamp.org/a-beginners-guide-to-website-optimization-2185edca0b72

I think for a long-term goal it would be appropriate to have both mobile and desktop optimization in the upper 90s... optimization is super important for new users especially those without great/stable Internet connections. 

For the future, going from 95 -> 99 without losing quality is probably harder than going from 50 -> 75 so I think bounties should reflect that if/when we decide to go that route :)

**Comment by owocki** [comment]
thanks for logging this issue..

i wonder if we want to bounty 'increase the pagespeed ranking by 10 points' or if we want to figure out what needs to be done and just bounty each of those individual tasks

**Comment by mkosowsk** [comment]
@owocki at your discretion, but think it's always interesting to get new folks into the project so think it might be interesting to 

0. Bounty individual tasks with the tasks relatively laid out for a beginner looking to make an impact on the project
1. Require that the task is completed by a NEW contributor to gitcoin/web (sorry @thelostone-mc but you need to take break 😂)

What do you think?

**Comment by owocki** [comment]
id be down to do that but i dont have the time today (or likely until much later in the week) to breakdown the tasks..  any chance you want to lay a few out?  happy to tip ya fer yer time

**Comment by mkosowsk** [comment]
Sure, will do before EOD

**Comment by mkosowsk** [comment]
@owocki created https://github.com/gitcoinco/web/issues/608 and https://github.com/gitcoinco/web/issues/609

Let me know what you think 👍🏻
